packages/react/package.json

```
@@ -44,7 +44,7 @@
   },
   "repository": {
     "type": "git",
-    "url": "",
+    "url": "https://github.com/VKCOM/mini-apps-analytics/tree/master/packages/react",
```

```suggestion
    "url": "https://github.com/VKCOM/mini-apps-analytics.git",
```

в **directory** уже указано `packages/react`

---

спасибо. поправил